src/githubHelper.ts

```diff
@@ -586,6 +785,11 @@ export default class GithubHelper {
     }
 
     // Failing all else, create an issue with a descriptive title
+
+    let userIsCreator = pullRequest.author && 
+      ((settings.usermap && settings.usermap[pullRequest.author.username] === settings.github.token_owner) ||
+       (pullRequest.author.username === settings.github.token_owner));
```

Comment:
You check that multiple times. Doesn't it make sense to extract it into its own function?

Reply:
yes, it is checked twice and I think I even factored it out into a function but it wasn't exactly the same functionality, then I simplified it and now it is indeed the same 😅 I'll fix that too.